manifests/example-template-windows.yml

```diff
@@ -0,0 +1,67 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: windows-vm-template
+  annotations:
+    description: "OCP kubevirt windows VM template"
+    tags: "kubevirt,ocp,template,windows"
+  labels:
+    miq.github.io/kubvirt/os: windows-2016
+    miq.github.io/kubevirt-is-vm-template: true
+objects:
+- apiVersion: kubevirt.io/v1alpha1
+  kind: OfflineVirtualMachine
+  metadata:
+    name: ${NAME}
+  spec:
+    template:
+      spec:
+        domain:
+          features:
+            acpi: {}
+            apic: {}
+            hyperv:
+              relaxed: {}
+              vapic: {}
+              spinlocks:
+                spinlocks: 8191
+          clock:
+            utc: {}
+            timer:
+              hpet:
+                present: false
+              pit:
+                tickPolicy: delay
+              rtc:
+                tickPolicy: catchup
+              hyperv: {}
+          cpu:
+            cores: ${CPU_CORES}
+          machine:
+            type: q35
+          resources:
+            requests:
+              memory: ${MEMORY}
+          devices:
+            disks:
+              - name: disk0
+                volumeName: disk0-pvc
+                disk:
+                  dev: sda
+        volumes:
+          - name: disk0-pvc
+            persistentVolumeClaim:
+              claimName: windows-vm-pvc-${NAME}
+- apiVersion: kubevirt.io/v1alpha1
+  kind: PersistentVolumeClaim
+  metadata:
+    name: windows-vm-pvc-${NAME}
```

Comment:
The clone annotation is missing - but is not blocking.

/cc @aglitke

Reply:
I am going to change the annotation to something that is independent of the storage provisioner being used but here is an example PVC with a clone request annotation:

```
kind: PersistentVolumeClaim
apiVersion: v1
metadata:
  name: test-clone
  annotations:
    standaloneCinderCloneRequest: golden-image
spec:
  storageClassName: standalone-cinder
  accessModes:
    - ReadWriteOnce
  resources:
    requests:
      storage: 1Gi
```

Reply:
Above pvc as defined in current example won't be accepted by kuberenetes. I asked to update it [here](https://github.com/ManageIQ/manageiq-providers-kubevirt/blob/master/docs/virtual_machine_template_guide.adoc) which aligns with Adam's suggestions. Can we update both examples?

Reply:
Sorry, it took me a bit too much to get up to speed here. Hopefully ok now.